src/main/kotlin/de/bund/digitalservice/useid/model/ClientResponseTCTokenUrl.kt

```diff
@@ -0,0 +1,6 @@
+package de.bund.digitalservice.useid.model
+
+data class ClientResponseTCTokenUrl(
```

Comment:
Can we find a better name? Something like `InitSessionResponse` or so would fit better, since it's not only the tc token url.